lib/cc/engine/flog.rb

```diff
@@ -128,24 +128,27 @@ def issue name, datum, location, score
           return
         end
 
-        remediation_points =
-          (BASE_REMEDIATION_POINTS + (OVERAGE_REMEDIATION_POINTS * score)).
-            round
-
```

Comment:
Not sure why this was refactored since it wasn't to test it directly and doesn't lend any extra clarity.